index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chef Curry</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<nav>
+    <div class="nav-bar">
+        <h1>Chef Curry's Kitchen</h1>
+        <ul class="nav-elements">
+            <a href="index.html">
+            <li>Home</li>
+            </a>
+            <li>About</li>
```

Comment:
It seems you forgot to include an about the chef page, can you please add this and make sure your navbar links to it.